src/odemis/acq/stream/_sync.py

```diff
@@ -151,6 +151,13 @@ def __init__(self, name, streams):
                 self._polarization = s.polarization
                 self._acquireAllPol = s.acquireAllPol
 
+        # Get integration time on detector if found in optical substream
```

Comment:
Let's more document _what_ integrationTime is supposed to mean here. Something like "Pick integrationTime if found on a stream (typically, optical). In this case, for each ebeam pos, the (short) acquisition will be repeated until the integration time is reached".